src/connection.rs

```diff
@@ -225,6 +225,44 @@ impl Connection {
         Ok(())
     }
 
+
+    #[cfg(feature = "encryption")]
+    #[inline]
+    pub fn encryption_key<T: AsRef<str>>(&self, data: T) -> Result<()> {
+        unsafe {
+            ok!(
+                self.raw.0,
+                ffi::sqlite3_key(
+                    self.raw.0,
+                    str_to_cstr!(data.as_ref()).as_ptr() as *const c_void,
+                    data.len() as c_int,
+                )
+            );
+        }
+        Ok(())
+    }
+
+    #[cfg(feature = "encryption")]
+    #[inline]
+    pub fn encryption_rekey<T: AsRef<str>>(&self, data: T) -> Result<()> {
+        unsafe {
+            ok!(
+                self.raw.0,
+                ffi::sqlite3_key(
```

Comment:
Type here.

Reply:
Should we switch to version 2?

https://www.zetetic.net/sqlcipher/sqlcipher-api/#sqlite3_rekey